templates/bot.mustache

```diff
@@ -1,168 +1,164 @@
 <!DOCTYPE html>
 <html>
-<head>
-{{> security }}
-
-{{> head }}
-	<title>Binary Bot</title>
-
-{{> fontello_css }}
-
-{{> bundle_css }}
-{{> bot_css }}
-{{> chart_css }}
-<link rel="stylesheet" href="https://style.binary.com/binary.css">
-
-{{> ie }}
-</head>
-<body>
-  <div id="center"></div>
-	<span id="restartTimeout"></span>
-	<div>
-		<div class="blocker"></div>
-		<div class="reality-check" style="display: none">
-			<h1 data-i18n-text="Reality Check"></h1>
-			<p data-i18n-text="Options trading can become a real addiction, as can any other activity pushed to its limits. To avoid the danger of such an addiction, we provide a reality-check that gives you a summary of your trades and accounts on a regular basis."></p>
-			<p data-i18n-text="Please specify your preferred reality-check interval in minutes">
-				<input id="realityDuration" interval="1" maxlength="4" min="1" size="6" type="number" value="10">
-			</p>
-			<p id="rc-err" style="display: none;" class="error-msg" data-i18n-text="Please enter a number between 10 to 120."></p>
-			<button bcont="1" class="button" id="continueTrading" data-i18n-text="Continue Trading"></button>
-		</div>
-	</div>
-  {{> loading }}
-  <div id="topbar">
-	  <div class="right-header">
-	    {{> language }}
-	    <select id="tours" style="display: none">
-	              <option value="" data-i18n-text="Select a Tour"></option>
-	      <option value="welcome" data-i18n-text="Welcome!"></option>
-	      <option value="introduction" data-i18n-text="Introduction"></option>
-	      <option value="makeSimpleStrategy" data-i18n-text="1. Make a simple strategy"></option>
-	    </select>
-	  </div>
-  </div>
-  <div id="header">
-      <div class="left-header">
-          <div class="logo-wrapper">
-              <a href="" id="logo">
-                  <div class="logo-parent">
-                      <div class="logo">
-                          <img class="responsive" src="https://style.binary.com/images/logo/symbol.svg" alt="Binary-logo" />
-                      </div>
-                      <div class="binary-logo-text">
-                          <img class="responsive" src="https://style.binary.com/images/logo/type.svg" alt="Binary-logo">
-                      </div>
-                  </div>
-              </a>
-          </div>
+  <head>
```

Comment:
@kellybinary It's always a good idea to pull the most recent changes from upstream  before you make a PR. Because sometimes changes from other collaborators will cause merge conflicts.
👍 for fixing the indentation.